fractal.config.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+/* Create a new Fractal instance and export it for use elsewhere if required */
+const fractal = module.exports = require('@frctl/fractal').create();
+
+fractal.set('project.version', 'v1.0');
+fractal.set('project.author', 'St. Jude Design System');
+fractal.set('project.title', 'design.stjude.cloud');
+
+/* Tell Fractal where the components will live */
+fractal.components.set('path', __dirname + '/components');
+
+/* Tell Fractal where the documentation pages will live */
+fractal.docs.set('path', __dirname + '/docs');
+
+/* Specify a directory of static assets */
+fractal.web.set('static.path', __dirname + '/dist');
+
+/* Set the static HTML build destination */
+fractal.web.set('builder.dest', __dirname + '/fractal-build');
+
+// require the Mandelbrot theme module
+const mandelbrot = require('@frctl/mandelbrot');
+
+// create a new instance with custom config options
+const myCustomisedTheme = mandelbrot({
+  skin: 'blue',
+  nav: ['search', 'docs', 'components', 'information'],
+  styles: [
+    'default',
+    '/css/fractal-theme.css'
+  ],
+  favicon: '/images/favicon.ico',
+  panels: ['html', 'resources', 'info', 'notes'], //'view' will hsow the handlebars template
```

Comment:
small typo in the comment. I'm not sure what 'view' is either but I'm guessing that's just because I haven't used fractal before.